### PR TITLE
chore(.travis.yml): fix e2e tests on Travis (run on latest Chrome)

### PR DESCRIPTION
**Pull request by gkalpak**
Previously, the tests would always fail/not run, because our current version of WebDriver requires a
newer version of Chrome than the one installed on the VM.

### CODE REVIEW

**Comment by gkalpak** [comment]
Some notes:

- Our current version of WebDriver requires Chrome 46 or newer.
- The default OS on Travis VMs is Ubuntu 12.04 (at the moment), which is not compatible with the latest Chrome (v51).
- This PR switches to beta Travis feature: Ubuntu Trusty (14.04), which (at the moment) _is_ compatible with the latest Chrome.
- This PR also introduces `directConnect: true` for running the tests with `scripts/private/test-all.sh`, which avoids having to specify the binary Chrome path on Travis and speeds up the tests.

--
Possible alternative solutions:

- Run the tests on Firefox.
- Use Travis' [docker support][1] to pull down and run the tests inside a container with a more recent OS version.
- Use an external provider, such as SauceLabs and/or BrowserStack.

[1]: https://docs.travis-ci.com/user/docker/

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
+sudo: required
```
Comment (gkalpak) [inline .travis.yml]: This is the default for repositories created before 2015. I am just making it explicit.
This is required for using Ubuntu 14.04 (Trusty).

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+dist: trusty
+
 language: node_js
 node_js:
   - '4.4'
 
-env: BROWSERS=firefox E2E_BROWSERS=Firefox
```
Comment (gkalpak) [inline .travis.yml]: This wasn't used anywhere.

File: .travis.yml [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
-
 before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-  - npm run update-webdriver
```
Comment (gkalpak) [inline .travis.yml]: This is being taken care of inside `test-all.sh`.

**Comment by petebacondarwin** [comment]
LGTM!!

**Pull request closed without merging**